subliminal/providers/legendastv.py

```diff
@@ -164,6 +164,13 @@ class LegendasTVProvider(Provider):
     subtitle_class = LegendasTVSubtitle
 
     def __init__(self, username=None, password=None):
+
+        # Provider needs UNRAR installed. If not available raise ConfigurationError
+        try:
+            rarfile._check_unrar_tool()
```

Comment:
@sharkykh from the docs both seem valid:

Problem reported by unrar/rar.
http://rarfile.readthedocs.io/en/latest/api.html#rarfile.RarExecError

Executable not found.
http://rarfile.readthedocs.io/en/latest/api.html#rarfile.RarCannotExec